uwdata/lyra
# Remove unused package deps & build scripts

**Description of the problem this pull request fixes**

Many dependencies specified in package.json are not in use within Lyra, and not all of the defined package scripts have the desired (or any) effect.

**Changes proposed**

This patch removes all NPM dependencies that are not obviously being used in the application.

It also removes the infinitely-recursing `npm test` script, and nixes the two `postbuild:` steps: the application is not currently set up to handle minified files, and we will be implementing minification via the webpack bundler rather than calling uglify directly when we do get to that point.

**Steps to functionally test**
1. Remove your `node_modules` directory
2. Reinstall the application
3. Confirm that Lyra builds
4. Load the dev server and confirm Lyra works as expected

LYRA-262, LYRA-263
